src/test/java/hw2/ex1/FirstScenarioTest.java

```diff
@@ -0,0 +1,203 @@
+import org.testng.annotations.Test;
+import org.testng.annotations.AfterTest;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class FirstScenarioTest {
+
+    private WebDriver driver;
+
+    @BeforeTest
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        this.driver = new ChromeDriver();
+        driver.manage().timeouts().
+                implicitlyWait(10000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                pageLoadTimeout(20000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                setScriptTimeout(30000, TimeUnit.MILLISECONDS);
+    }
+
+    @Test
+    public void firstScenarioTest() {
+        // 1. Open test site by URL
+        // There's a simple opening of a site
+        driver.get("https://epam.github.io/JDI/index.html");
+
+        // 2. Assert Browser title
+        // There's a getting of a title and
+        // comparing it with expected title -- "Home page"
+        String actualTitle = driver.getTitle();
+        assertEquals(actualTitle, "Home Page");
+
+        // 3. Perform login
+        // There we have a simple login steps:
+        // - clicking on icon
+        // - entering username and password
+        // - clicking on a login-button
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        // 4. Assert User name in the left top side
+        // of screen that user is logined
+        // There we have a getting of username and
+        // comparing it with expected -- "PITER CHAILOVSKII"
+        // (We should compare it as is - even if it's
+        // written in upper case
+        WebElement actualUserName = driver.findElement(By.id("user-name"));
+        assertEquals(actualUserName.getText(), "PITER CHAILOVSKII");
+
+        // 5. Assert Browser title
+        // There's a getting of a title and
+        // comparing it with expected title -- "Home page"
+        // NB! We take expectedTitle from above
+        actualTitle = driver.getTitle();
+        assertEquals(actualTitle, "Home Page");
+
+        // 6. Assert that there are 4 items on the header
+        // section are displayed and they have proper texts
+        // There's a creating of array list with expected texts
+        // in header, getting a web element with actual header
+        // texts, getting list of these texts and comparing them
+        ArrayList<String> expectedHeaderTexts = new ArrayList<>();
+        expectedHeaderTexts.add("HOME");
+        expectedHeaderTexts.add("CONTACT FORM");
+        expectedHeaderTexts.add("SERVICE");
+        expectedHeaderTexts.add("METALS & COLORS");
```

Comment:
This peace of code could be simplified to the one line :)

Reply:
Solved: now I use *Arrays.asList(. . .)* instead of this.